server/issue.go

```diff
@@ -1425,26 +1434,6 @@ func (s *Server) postInboxIssueActivity(ctx context.Context, issue *api.Issue, a
 	return nil
 }
 
-func (s *Server) getTenantDatabaseMatrix(ctx context.Context, projectID int, dbNameTemplate string, dbList []*api.Database, baseDatabaseName string) ([]*api.Deployment, [][]*api.Database, error) {
```

Comment:
Why do we choose to remove this shared function and duplicate the code in 2 places?

Reply:
Updated the PR description a bit. And updated the PR to make it clear that deploySchedule will be used throughout getPipelineCreateForDatabaseSchemaAndDataUpdate().